docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -72,3 +72,18 @@ items:
       kubernetes.io/hostname: ocp-worker-0
       node-role.kubernetes.io/worker: ""
 ```
+
+# TODO: Move this to the IPv6 section once it is fully documented, both upstream and downstream.
```

Comment:
I think asciidoc uses `//` for comments rather than `#`.

Reply:
You are totally right, sorry for that! I updated it